src/geocoder/index.js

```diff
@@ -6,11 +6,13 @@ import type { Feature } from '../../flow-types/Feature'
 import type { Coordinates } from '../../flow-types/Coordinates'
 
 type PositionParam = {
-    'focus.point.lat': number,
-    'focus.point.lon': number
-}
+    "focus.point.lat": number,
+    "focus.point.lon": number
+};
```

Comment:
Please remove all semi colons. `npm run lint -- --fix` should take a lot of them.